.github/workflows/pr-integration.yml

```diff
@@ -96,8 +96,15 @@ jobs:
         test-server: workspace-local.json
         test: suites/FullIntegration.json
 
+    - name: Skip version bump merges
+      id: bump-check
+      uses: ./.github/actions/bump-skip
+      with:
+        event-name: ${{ github.event_name }}
+
     - name: "Notify WSM Slack"
-      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main'
+      # post to WSM Slack when a regular push (i.e. non-bumper push) is made to main branch
+      if: always() && github.event_name == 'push' && github.ref == 'refs/heads/main' && steps.bump-check.outputs.is-bump == 'no'
```

Comment:
I do not think you need the `github.ref == 'refs/heads/main'` condition. Testing for `'push'` should be sufficient.

Reply:
Oh yeah, that's true, since the `on push` trigger is only targeted for main branch.